Helper/Quote.php

```diff
@@ -72,8 +72,18 @@ public function __construct(
 
     public function sendQuote(\Magento\Quote\Model\Quote $quote, \Drip\Connect\Model\Configuration $config, string $action)
     {
+      $items = [];
+      foreach ($quote->getAllItems() as $item) {
+          $itemData = [
```

Comment:
Nit: You can drop the local variable.
```suggestion
          $items[] = [
```